timeboost-sequencer/src/lib.rs

```diff
@@ -190,12 +191,17 @@ impl Sequencer {
 
             let rcf =
                 RbcConfig::new(cfg.sign_keypair.clone(), committee.clone()).recover(cfg.recover);
-            let rbc = Rbc::new(Overlay::new(net), rcf.with_metrics(rbc_metrics));
+
+            let rbc = Rbc::new(
+                5 * committee.size().get(),
```

Comment:
what is 5 here?

Reply:
A magic number. The exact capacity should not matter.